src/api/v1beta1/properties.go

```diff
@@ -63,6 +65,10 @@ func (dk *DynaKube) NeedsOneAgent() bool {
 	return dk.ClassicFullStackMode() || dk.CloudNativeFullstackMode() || dk.HostMonitoringMode()
 }
 
+func (dk *DynaKube) OneAgentDaemonsetName() string {
+	return fmt.Sprintf("%s-%s", dk.Name, PodNameOSAgent)
+}
```

Comment:
I'd suggest adding a unit test, which checks if this property is calculated correctly. Then the property can be used in other unit tests without having to manually concatenating it.

A very simple one would suffice. E.g.:
```
func TestOneAgentDaemonsetName(...) {
  instance := &DynaKube{
    ObjectMeta: {
      Name: "test",
    },
  }
  assert.Equal("test-oneagent", instance.OneAgentDaemonsetName())
}
```

Reply:
Ah, yeah, good idea. Haven't though about that. 👍